3drepo/3drepo.io
# Issue 4908 - Milestone 3: Federation/container selection for calibration

This fixes #4908 

#### Description
Drawing button is now a dropdown and should act as follows (<name>: [<options>]):
 - Empty: _dropdown is disabled_;
 - Calibrated (green): [Recalibrate];
 - Uncalibrated: [Calibrate];
 - !Calibrated (orange): [Approve calibration, Recalibrate];
> Approve calibration updates the state to Calibrated (green)

#### Test cases
- Go to the drawings list in the dashboard and select any calibrate/recalibrate options. This should pop up a model selection modal and lead to the calibration view
- Open the viewer and select any calibrate options. This should lead to the calibration view using the current model
